resources/cygnus-translators/README.md

```diff
@@ -132,7 +133,59 @@ $ hadoop fs -cat /user/johndoe/dataset/data.txt
 
 [Top](#top)
 
-##<a name="section5"></a>Reporting issues and contact information
+##<a name="section5"></a>`cygnus_translator_pre0.10.0_to_0.10.0_mysql.sh`
+This scripts adds certain fields not available in previous versions to 0.10.0 to the MySQL tables potentially containing historical context data. Specifically:
```

Comment:
certain field -> the exact fields that the script is adding (to be more clear and precise)